content/docs/command-reference/remote/modify.md

```diff
@@ -586,16 +586,31 @@ for a full guide on using Google Drive as DVC remote storage.
   ```
 
 - `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
-  access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
+  access Google Drive, by default
+  `$CACHE_HOME/pydrive2fs/{gdrive_client_id}/default.json` (see also
+  `profile` below), where the`CACHE_HOME` location per platform is:
+
+  | macOS              | Linux  (\*typical) | Windows                 |
+  | ------------------ | ------------------ | ----------------------- |
+  | `~/Library/Caches` | `~/.cache`         | `%CSIDL_LOCAL_APPDATA%` |
 
   ```dvc
   $ dvc remote modify myremote \
-        gdrive_user_credentials_file .dvc/tmp/mycredentials.json
+        gdrive_user_credentials_file path/to/mycredentials.json
   ```
 
 See [Authorization](/doc/user-guide/setup-google-drive-remote#authorization) for
 more details.
 
+- `profile` - name for the credentials cache (part of the default
+  `gdrive_user_credentials_file` path). By default, credentials will be cached
+  per `gdrive_client_id `, but if you have multiple system users, you can set a
+  custom ‘profile’ name to avoid using someone else's credentials accidentally.
```

Comment:
```suggestion
- `profile` - name for the GDrive credentials cache directory (part of the
  global `gdrive_user_credentials_file` path). By default, credentials are cached
  per `gdrive_client_id `, but you can set a custom ‘profile’ instead, for example
  to avoid using the wrong user's credentials accidentally.
```